lib/usd/hdMaya/adapters/proxyAdapter.h

```diff
@@ -1,15 +1,15 @@
 #ifndef HDMAYA_AL_PROXY_ADAPTER_H
 #define HDMAYA_AL_PROXY_ADAPTER_H
 
-#include "../delegates/proxyUsdImagingDelegate.h"
-#include "shapeAdapter.h"
+#include <hdMaya/delegates/proxyUsdImagingDelegate.h>
+#include <hdMaya/adapters/shapeAdapter.h>
```

Comment:
Swap these two so "adapters" is before "delegates"?

Reply:
see 7cb7810a